docs/topics/snippet-reports-filter-sort.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: SNIPPET
+
+
+.Procedure
+
+. In *Migration* view, click *Reports*.
+. In the *Current landscape* pane, select *All questionnaires* or a single questionnaire from the drop-down list to see an overview of the risk distribution (High, Medium, Low, Unassessed/Unknown).
+. In the *Identified risks* pane, filter the report results by selecting from the drop-down list: Questionnaire, Section, Question, Answer, or Risk.
```

Comment:
I so want to be pedantic and say **Identified risks's** but I will be well-behaved

Reply:
That's the name of the pane   
;-)